server/database/schema.sql

```diff
@@ -445,17 +469,13 @@ create table lab12_exercise
 );
 
 INSERT INTO public.labs (id, "labName", "labShortName", category, "thumbnailImageURL", "shortDescription", "fullDescription", "learningObjectives", authors, "labURL", "copyrightAttributes", about, reading, reinforcement, quiz, difficulty, "slideshow", "walkthroughVideo", "isActive")
-VALUES (0, 'Building Labs with ALL',
-        'Building Labs with ALL', 'Accessibility', '/wrench.jpg',
+VALUES (0, 'How to Design a Lab', 'How to Design a Lab', 'Accessibility', '/wrench.jpg',
```

Comment:
I wonder if we might want to create a new category for this lab, like 'Tutorial' or something like that. What do you think?